Resource/PHPCSFixer/Fixer/OpeningTag.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Hoa\Devtools\Resource\PHPCsFixer\Fixer;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\FixerInterface;
+use Symfony\CS\Tokenizer\Tokens;
+use Symfony\CS\Tokenizer\Token;
+use SplFileInfo;
+
+/**
+ * Ensure there is a new line after the <php.
+ */
+class OpeningTag extends AbstractFixer
+{
+    public function fix(SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        if (!$tokens->isMonolithicPhp()) {
+            return $content;
+        }
+
+        if (!$tokens[0]->isGivenKind(T_OPEN_TAG)) {
+            return $content;
+        }
+
+        if (!$tokens[1]->isWhitespace()) {
+            $tokens[0]->setContent('<?php');
+            $tokens->insertAt(1, new Token([T_WHITESPACE, "\n\n"]));
```

Comment:
Ok to optimize it but it's different of `blankline_after_open_tag`. For instance, if we have:
```php
<?php /* …
```
the `blankline_after_open_tag` will do nothing as far as I know, or at least, it will transform it into:
```
<?php
/* …
```
and we expect
```php
<?php

/*
…
```

Any idea how to not “corrupt” the token collection? I don't even know why it corrupts? Because of the `insertAt`?